AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -246,6 +253,8 @@ def camera(property: str = 'name') -> str:
     def cube_map_capture(property: str = 'name') -> str:
         """
         CubeMap capture component properties.
+        :param property: From the last element of the property tree path. Default 'name' for component name string.
+        :return: Full property path OR component name if no property specified.
```

Comment:
Why are you updating cube_map_capture function within the hair PR?

Didn't think hair had anything to doo with cube maps...

Reply:
just fixing something I missed in review. the new cube_map_capture didn't specify parameters or return type.

Reply:
gotcha, resolving